packages/i18n-calypso/package.json

```diff
@@ -13,25 +13,20 @@
 	},
 	"author": "@automattic",
 	"license": "GPL-2.0-or-later",
-	"bin": {
-		"i18n-calypso": "cli/index.js"
```

Comment:
In the readme file? :-)

I don't think there's technical justification to add to `optionalDependencies`/`peerDependencies` since the package really doesn't depend on the CLI?

Reply:
Well, it's the only tool that can extract translatable strings wrapped in `translate()` calls, no?

Reply:
Still not a super-hard dep IMO. Like it's not required for `i18n-calypso` to work on a readily string-extracted source.

Reply:
`package.json` dependencies are not for humans, but for machines to figure out the NPM dependency tree. But we can add a section in README.